bap_ida_python/loader/installer.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python
+"""Loader for setting up IDA to successfully import all BAP plugins."""
+
+
+def _multi_line_string_clean(s):
+    s_split = s.split('\n')[1:]
+    for i, c in enumerate(s_split[0]):
+        if not c.isspace():
+            left_space_count = i
+            break
+    return '\n'.join(x[left_space_count:] for x in s_split)
+
+
+def get_bap_path():
+    """Return path of bap executable or None."""
+    from subprocess import check_output, CalledProcessError
+    import os
+    try:
+        bap_path = check_output(['which', 'bap']).strip()
+    except OSError:  # Cannot run 'which' command
+        print "[!] Could not determine bap path automatically."
+        print "[ ] Please enter path to bap executable (including /bap): "
+        bap_path = input('  BAP Path: ').strip()
+    except CalledProcessError:  # 'which' could not find 'bap'
+        print "[!] Unable to locate bap. Please install it and ensure that it"
+        print "        is on your path, and `bap --version` works."
+        return None
+    if os.path.isfile(bap_path) and bap_path.endswith('/bap'):
+        return bap_path
+    else:
+        print "[!] Unable to locate bap at {}".format(bap_path)
+        return None
+
+
+def install_bap_config():
```

Comment:
Due to the non-standard path of `bap` and non-standard path of `ida`, it is going to be necessary to use something to communicate what is where.

I have not added it on a per-need basis, and have spent quite some time thinking about how the config would be work. And that is why I have chosen the format `(BAP (k1 v1) (k2 v2) (k3 v3) ...)` (i.e. an S-Expression starting with `BAP`) as the format. Please suggest alternatives if you find necessary.

Also, any suggestions for how to pass the info better would definitely be appreciated :)